doc/reference_accessor.xml

```diff
@@ -1740,8 +2122,8 @@ SELECT * FROM ST_IsValidDetail('LINESTRING(220227 150406,2220227 150407,222020 1
 	  <refsection>
 		<title>Description</title>
 
-		<para>Return the M coordinate of the point, or NULL if not
-			available. Input must be a point.</para>
+		<para>Return the M crdinate of a Point, or NULL if not
```

Comment:
as a non-native English speaker it's painful to learn word "ordinate". they're all "coordinates" in Russian, "ordinate" is specifically Y axis in version of maths I was taught - "abscissa" is for X. Is "M ordinate" valid thing in English, or can we keep "M coordinate" which also matches Russian usage?
```suggestion
		<para>Return the M ordinate of a Point, or NULL if not
```

Reply:
Ok, good to know.